services/app/dockers/cpp/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM alpine:3.4
+FROM debian:9-slim
 
-RUN apk add --update make
-RUN apk add --no-cache git openssh-client g++ cmake ninja python3 wget
+RUN apt-get update && \
+    apt-get install -y git openssh-client g++ cmake ninja-build python3 wget cppcheck clang-tidy && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
 
```

Comment:
Нужно подправить под требования СI
https://app.codacy.com/gh/hexlet-codebattle/codebattle/pullRequest?prid=4970667

Reply:
И билд полностью рухнул https://github.com/hexlet-codebattle/codebattle/pull/629/checks?check_run_id=434490478
@vtm9 Может на стороне githab actions проблема?

Reply:
@ReDBrother надо ли указывать версии пакетов устанавливаемых через `apt install`?

Reply:
Это скорее предосторожность. Ею можно пренебречь.

Reply:
Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>` Блин, про это линтер и пишет. Значит придется указывать.